ui-participant/src/hub/kit/KitInstructions.tsx

```diff
@@ -1,162 +1,146 @@
+    </div>
+  </div>
+}
 
-              <KitCollectionStepWrapper title={'Step 3: Select return option'}
-                status={selectedReturnType ? 'COMPLETE' : 'INCOMPLETE'}>
-                <div className='mb-3'>
-                  Once you have completed the kit, you may return it to the study
-                  team in-person, or ship it back using the provided return label.
-                </div>
-                <div className={'fw-bold mb-2'}>Select a return option:</div>
-                <div className="form-check">
-                  <input className="form-check-input" type="radio" name="returnOption" id="returnInPerson"/>
-                  <label className="form-check-label" htmlFor="returnInPerson"
-                    onClick={() => setSelectedReturnType('IN_PERSON')}
-                  >
-                    Return to study team in-person
-                  </label>
-                </div>
-                <div className="form-check">
-                  <input className="form-check-input" type="radio" name="returnOption" id="returnViaMail"/>
-                  <label className="form-check-label" htmlFor="returnViaMail"
-                    onClick={() => setSelectedReturnType('RETURN_LABEL')}
-                  >
-                    Return via mail using provided return label
-                  </label>
-                </div>
-              </KitCollectionStepWrapper>
+const KitContent = ({ enrollee }: { enrollee: Enrollee }) => {
+  const activeKit = enrollee.kitRequests.filter(kit => kit.kitOriginType === 'ASSIGNED')[0]
 
-              {selectedReturnType === 'IN_PERSON' &&
-                <KitCollectionStepWrapper title={'Step 4: Return sample'}
-                  disabled={!selectedReturnType}
-                  status={activeKit ? 'COMPLETE' : 'INCOMPLETE'}>
-                  <div className='mb-3'>
-                    A member of the study team will scan the code below along with the
-                    barcode on the sample kit to confirm sample receipt.
-                  </div>
-                  <div className="d-flex flex-column align-items-center">
-                    <QRCode value={activeEnrollee?.shortcode} size={200} className={'pb-3'}/>
-                  </div>
-                  <div className="pb-3">
-                    You will receive an email confirmation once your sample collection has been successfully
-                    associated with your account, and you can view the status of your kit on the
-                    <Link to='/hub'> participant dashboard.</Link>
-                  </div>
-                  <div className="d-flex justify-content-center">
-                    <Link to={'/hub'} className="btn rounded-pill ps-4 pe-4 fw-bold btn-primary">
-                      Return to Dashboard
-                    </Link>
-                  </div>
-                </KitCollectionStepWrapper>
-              }
+  if (!enrollee.consented) {
```

Comment:
Minor nit: I feel like this component might be a bit more readable if each of these branches was its own component